### PR TITLE
edgeTransport 2.0 release

### PR DESCRIPTION
**Pull request by johannah-pik**
- Refactor and adjust the read, covert, calc functions used to produce transport input data for REMIND and the iterative edgeTransport version
- calcEDGETrData was removed
- clean readEDGETransport(), convertEDGETransport() and calcEDGETRansport() functions were introduced
- weight of f35_esCapCost [2005US$/(p|t)km] was changed from gdp to energy service demand [(p|t)km]
- weight of f35_fe2es ["trn (p|t)km/Twa"] was changed to final energy [TWa]

This PR is related to the edgeTransport 2.0 release and merged together with the following PRs:
mrtransport [#13](https://github.com/pik-piam/mrtransport/pull/13)
edgeTransport [#266](https://github.com/pik-piam/edgeTransport/pull/266/files)
reporttransport [#18](https://github.com/pik-piam/reporttransport/pull/18)
remind [#1749](https://github.com/remindmodel/remind/pull/1749)
remind2 [#642](https://github.com/pik-piam/remind2/pull/624)
piamInterfaces [#331](https://github.com/pik-piam/piamInterfaces/pull/331)

### CODE REVIEW

File: R/calcEU_ReferenceScenario.R [hunk 1]
```diff
@@ -12,7 +12,7 @@ calcEU_ReferenceScenario <- function() {
 
   weights <- x
   weights[, , ] <- NA
-  weights[, , "Price|Secondary Energy|Electricity (US$2005/GJ)"] <- 1
+  weights[, , "Price|Secondary Energy|Electricity (EUR2013/GJ)"] <- 1
```
Comment (johannah-pik) [inline R/calcEU_ReferenceScenario.R:15]: @fbenke-pik is this bugfix correct and shall we include it here or separately?

Comment (fbenke-pik) [reply R/calcEU_ReferenceScenario.R:15]: Yes, this is correct. Thank you for fixing.

**Review by jmuessel — APPROVED** [review]

File: R/fullREMIND.R [hunk 2]
```diff
@@ -16,36 +18,32 @@ fullREMIND <- function() {
   rem_years <- seq(2005, 2150, 5)
   rem_years_hist <- seq(1990, 2150, 5)
 
-  # Enable madrat caching for edgeTransport
-  madrat::madratAttach("edgeTransport")
+  madratAttach("edgeTransport")   # enable madrat caching for edgeTransport
```
Comment (LaviniaBaumstark) [inline R/fullREMIND.R:21]: please add madrat::: @fbenke-pik do you agree?

File: R/fullREMIND.R [hunk 3]
```diff
@@ -135,22 +134,6 @@ fullREMIND <- function() {
   calcOutput("CCScapacity", subtype = "projections",                   round = 8,  file = "pm_boundCapCCS.cs4r")
   calcOutput("CCSbounds",                                              round = 8,  file = "p_boundCapCCSindicator.cs4r")
   calcOutput("LimitCCS",                                               round = 8,  file = "pm_dataccs.cs3r")
-  calcOutput('Industry_CCS_limits',
```
Comment (fbenke-pik) [inline R/fullREMIND.R:138]: Was this deleted intentionally?

File: R/fullREMIND.R [hunk 4]
```diff
@@ -118,9 +116,10 @@ fullREMIND <- function() {
   calcOutput("ShareIndFE",                            round = 3,  file = "p37_shIndFE.cs3r")
   calcOutput("nonEnergyIndFE",                        round = 8,  file = "f37_fedemand_NonEnergyIndst.cs4r")
   calcOutput("Clinker_to_cement_ratio",               round = 2,  file = "p37_clinker-to-cement-ratio.cs3r")
-  # Delete the 'dummy' line
+  # delete the 'dummy' line
   system(paste0('sed -i "/dummy/d" ', getConfig()$outputfolder, "/p37_clinker-to-cement-ratio.cs3r"))
 
+  calcOutput("Capacity", subtype = "capacityByTech",                   round = 6,  file = "p_histCap.cs3r")  # will be deleted after the merge of REMIND-EU
```
Comment (fbenke-pik) [inline R/fullREMIND.R:122]: Was this brought back intentionally?  @jmuessel

Comment (jmuessel) [reply R/fullREMIND.R:122]: I think we needed this for pCap variables later. I would leave this in for now. But I noted it to check again in some weeks, when we do sth on pCap variables.

File: R/fullREMIND.R (hunk 3 above)
Comment (jmuessel) [reply R/fullREMIND.R:138]: No, I'll a PR rn

File: R/fullREMIND.R (hunk 4 above)
Comment (fbenke-pik) [reply R/fullREMIND.R:122]: why can't you use the file `pm_histCap.cs3r` (see below)

Comment (jmuessel) [reply R/fullREMIND.R:122]: I think you are right but I am not sure atm.

**Pull request merged**